src/ert/_c_wrappers/enkf/config/enkf_config_node.py

```diff
@@ -193,6 +188,10 @@ def get_init_file_fmt(self) -> str:
     def getObservationKeys(self) -> StringList:
         return self._get_obs_keys().setParent(self)
 
+    @classmethod
+    def setForwardInit(cls, forward_init: bool):
+        cls._forward_init = forward_init
```

Comment:
Since this is a class instance, I would imagine that this is shared across all instances? Isn't it necessary to use an instance attribute?

Reply:
This is long gone in the final version.